src/languageSupport/languages/agnostic/connection.ts

```diff
@@ -0,0 +1,183 @@
+import * as MonacoTypes from 'monaco-editor/esm/vs/editor/editor.api'
+import isEqual from 'lodash/isEqual'
+
+import {
+  DEFAULT_SELECTION,
+  CompositionSelection,
+} from 'src/dataExplorer/context/persistance'
+
+// Types
+import {EditorType, RecursivePartial} from 'src/types'
+import {LspRange as FluxLspRange} from 'src/languageSupport/languages/flux/lsp/types'
+
+// Utils
+import {notify} from 'src/shared/actions/notifications'
+import {oldSession} from 'src/shared/copy/notifications'
+
+// May as well re-use this well formed type.
+export type LspRange = FluxLspRange
```

Comment:
Because the `LspRange` should be language agnostic. Since the datePicker is no longer specifically for use in flux, and here is being used in SQL.

Instead of doing the work (in this PR) to rename all the classnames, type names, etc affliated with the datePicker to be not `flux--<rest>` --- we are instead just mapping the `FluxLspRange` type to the agnostic type. 

In a future cleanup PR, we can choose to make the datePicker not longer refer to `flux--` anywhere.

Reply:
done